src/components/webview/FileExplorerProvider.ts

```diff
@@ -38,14 +39,15 @@ import {
 	compareCaseElements,
 } from '../../elements/buildCaseElement';
 import { CaseManager } from '../../cases/caseManager';
+import { DiffWebviewPanel } from './DiffWebviewPanel';
 
 export class FileExplorerProvider implements WebviewViewProvider {
 	__view: WebviewView | null = null;
 	__extensionPath: Uri;
 	__webviewResolver: WebviewResolver | null = null;
 	__elementMap = new Map<ElementHash, Element>();
 	__folderMap = new Map<string, TreeNode>();
-	__fileNodes = new Set<TreeNode>();
+	__fileNodes = new Map<string, { jobHash: JobHash; node: TreeNode }>();
```

Comment:
is this a map between URIs to jobs?

Reply:
It's a map between URIs to the File Tree Node and the job hash.

Left it as a comment